engine/access/rpc/backend/backend_scripts.go

```diff
@@ -14,12 +16,16 @@ import (
 	"github.com/onflow/flow-go/storage"
 )
 
+// uniqueErrorLogTimeWindow is the duration for checking the uniqueness of scripts sent for execution
+const uniqueErrorLogTimeWindow = time.Duration(10) * time.Minute
```

Comment:
nit: since this has nothing to do with error could we rename it to `unqiueScriptLoggingTimeWindow` or something to that effect?